portfolio/src/main/webapp/piano_im.js

```diff
@@ -0,0 +1,49 @@
+const notes = new Map();
+notes.set('C', 'white');
+notes.set('C#', 'black');
+notes.set('D', 'white');
+notes.set('D#', 'black');
+notes.set('E', 'white');
+notes.set('F', 'white');
+notes.set('F#', 'black');
+notes.set('G', 'white');
+notes.set('G#', 'black');
+notes.set('A', 'white');
+notes.set('A#', 'black');
+notes.set('B', 'white');
+notes.set('C2', 'white');
+notes.set('C#2', 'black');
+notes.set('D2', 'white');
+notes.set('D#2', 'black');
+notes.set('E2', 'white');
+
+const svgns = 'http://www.w3.org/2000/svg';
+const svg = document.getElementsByTagName('svg')[0];
+const whiteWidth = 100;
+const blackWidth = 30;
+const whiteHeight = '300';
```

Comment:
whiteHeight and blackHeight are strings here - should they be const, and defined as integers instead of strings? This way it's consistent with the integer widths in the lines above.